plugins/multi-chain-ecdsa/actions/sendUserOperations.ts

```diff
@@ -0,0 +1,345 @@
+import {
+    AccountNotFoundError,
+    type Action,
+    type KernelSmartAccountImplementation,
+    KernelV3AccountAbi,
+    getEncodedPluginsData,
+    isPluginInitialized
+} from "@zerodev/sdk"
+import { MerkleTree } from "merkletreejs"
+import {
+    type Chain,
+    type Client,
+    type Hash,
+    type Hex,
+    type Transport,
+    concatHex,
+    encodeAbiParameters,
+    getAbiItem,
+    hashTypedData,
+    keccak256,
+    toFunctionSelector,
+    zeroAddress
+} from "viem"
+import {
+    type PrepareUserOperationParameters,
+    type SendUserOperationParameters,
+    type SmartAccount,
+    type UserOperation,
+    getUserOperationHash,
+    prepareUserOperation,
+    sendUserOperation
+} from "viem/account-abstraction"
+import { parseAccount } from "viem/accounts"
+import { getAction } from "viem/utils"
+
+export type ClientWithChainId<
```

Comment:
Why do we need ClientWithChainId? Couldn't we just get it from original client like this `client.chain.id` ?